graphql-database-dynmodb-history-lambda/src/main/java/com/fleetpin/graphql/database/dynamo/history/lambda/DynamoUtil.java

```diff
@@ -0,0 +1,97 @@
+package com.fleetpin.graphql.database.dynamo.history.lambda;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+import com.amazonaws.services.lambda.runtime.events.DynamodbEvent.DynamodbStreamRecord;
+
+import software.amazon.awssdk.core.SdkBytes;
+import software.amazon.awssdk.services.dynamodb.model.AttributeValue;
+import software.amazon.awssdk.services.dynamodb.model.Identity;
+import software.amazon.awssdk.services.dynamodb.model.Record;
+import software.amazon.awssdk.services.dynamodb.model.StreamRecord;
+
+public class DynamoUtil {
+
+	public static Record toV2(DynamodbStreamRecord record) {
+
+		var builder = Record.builder();
+		builder.awsRegion(record.getAwsRegion());
+		builder.dynamodb(toV2(record.getDynamodb()));
+		builder.eventID(record.getEventID());
+		builder.eventName(record.getEventName());
+		builder.eventSource(record.getEventSource());
+		builder.eventVersion(record.getEventVersion());
+		builder.userIdentity(toV2(record.getUserIdentity()));
+		return builder.build();
+	}
+
+	public static Identity toV2(com.amazonaws.services.dynamodbv2.model.Identity userIdentity) {
+		var builder = Identity.builder();
+		builder.principalId(userIdentity.getPrincipalId());
+		builder.type(userIdentity.getType());
+		return builder.build();
+	}
+
+	public static StreamRecord toV2(com.amazonaws.services.dynamodbv2.model.StreamRecord dynamodb) {
+		var builder = StreamRecord.builder();
+
+		builder.approximateCreationDateTime(dynamodb.getApproximateCreationDateTime().toInstant());
+		builder.keys(toV2(dynamodb.getKeys()));
+		builder.newImage(toV2(dynamodb.getNewImage()));
+		builder.oldImage(toV2(dynamodb.getOldImage()));
+		builder.sequenceNumber(dynamodb.getSequenceNumber());
+		builder.sizeBytes(dynamodb.getSizeBytes());
+		builder.streamViewType(dynamodb.getStreamViewType());
+
+		return builder.build();
+	}
+
+	public static Map<String, AttributeValue> toV2(
+			Map<String, com.amazonaws.services.dynamodbv2.model.AttributeValue> attribute) {

```

Comment:
Don't need to use the fully-qualified class name.

Reply:
one on the left is software.amazon different classes

Reply:
ah yep, didn't notice that, should've been obvious from the behaviour of the method oops.